lib/spack/spack/test/cmd/load.py

```diff
@@ -125,6 +125,20 @@ def test_load_first(install_mockery, mock_fetch, mock_archive, mock_packages):
     load("--sh", "--first", "libelf")
 
 
+def test_load_best(install_mockery, mock_fetch, mock_archive, mock_packages):
+    """Test with and without the --best option"""
+    install("libelf@0.8.12 arch=x86_64")
+    install("libelf@0.8.12")
```

Comment:
The test here concretized to the host arch from a potentially different family, so I think it needs to mock `archspec.cpu.host`